src/main/java/net/porillo/util/CO2Notifications.java

```diff
@@ -36,13 +36,13 @@
 public class CO2Notifications {
     @Getter
     private Map<String, BossBar> bossBars;
-    private static final long NOTIFICATION_INTERVAL_TICKS = 6000; //5 minutes
-    private static final long NOTIFICATION_DURATION_TICKS = 300; //15 seconds
-    private static final int NORMAL_SEA_LEVEL = 63;
-    private static final double NORMAL_ICE_LEVEL_HEIGHT = 0.0;
-    private static final double NORMAL_SNOW_LEVEL_HEIGHT = 0.0;
-    private static final double NORMAL_FARM_YIELD_FITNESS = 100.0;
-    private static final double NORMAL_MOB_FITNESS = 100.0;
+    private static final long NOTIFICATION_INTERVAL_TICKS = GlobalWarming.getInstance().getConf().getNotificationInterval();
```

Comment:
Same comment as earlier, do we know if this values update properly after a /gw reload?